resources/schema/Operations.json

```diff
@@ -0,0 +1,338 @@
+      "properties": {
+        "operator": { "const": "required_variables" }
+      },
+      "required": ["id", "operator"],
+      "type": "object"
+    },
+    {
+      "properties": {
+        "operator": { "const": "study_domains" }
+      },
+      "required": ["id", "operator"],
+      "type": "object"
+    },
+    {
+      "properties": {
+        "operator": {
+          "const": "valid_codelist_dates"
+        }
+      },
+      "required": ["id", "operator"],
+      "type": "object"
+    },
+    {
+      "properties": {
+        "operator": {
+          "const": "valid_meddra_code_references"
+        }
+      },
+      "required": ["id", "operator"],
+      "type": "object"
+    },
+    {
+      "properties": {
+        "operator": {
+          "const": "valid_meddra_code_term_pairs"
+        }
+      },
+      "required": ["id", "operator"],
+      "type": "object"
+    },
+    {
+      "properties": {
+        "operator": {
+          "const": "valid_meddra_term_references"
+        }
+      },
+      "required": ["id", "operator"],
+      "type": "object"
+    },
+    {
+      "properties": {
+        "operator": {
+          "const": "valid_whodrug_references"
+        }
+      },
+      "required": ["id", "operator", "name"],
+      "type": "object"
+    },
+    {
+      "properties": {
+        "operator": { "const": "variable_count" }
+      },
+      "required": ["id", "operator"],
+      "type": "object"
+    },
+    {
+      "properties": {
+        "operator": { "const": "variable_exists" }
+      },
+      "required": ["id", "operator"],
+      "type": "object"
+    },
+    {
+      "properties": {
+        "operator": { "const": "variable_is_null" }
+      },
+      "required": ["id", "operator"],
+      "type": "object"
+    },
+    {
+      "properties": {
+        "operator": { "const": "variable_names" }
+      },
+      "required": ["id", "operator"],
+      "type": "object"
+    },
+    {
+      "properties": {
+        "operator": {
+          "const": "variable_permissibilities"
+        }
+      },
+      "required": ["id", "operator"],
+      "type": "object"
+    },
+    {
+      "properties": {
+        "operator": {
+          "const": "variable_value_count"
+        }
+      },
+      "required": ["id", "operator", "name"],
+      "type": "object"
+    },
+    {
+      "properties": {
+        "operator": {
+          "const": "whodrug_code_hierarchy"
+        }
+      },
+      "required": ["id", "operator"],
+      "type": "object"
+    }
+  ],
+  "properties": {
+    "attribute_name": {
+      "$ref": "MetaVariables.json"
+    },
+    "domain": {
+      "anyOf": [
+        {
+          "$ref": "CORE-base.json#/$defs/Dataset"
+        },
+        {
+          "$ref": "CORE-base.json#/$defs/DataStructure"
+        }
+      ]
+    },
+    "group": {
+      "items": {
+        "$ref": "CORE-base.json#/$defs/VariableName"
+      },
+      "type": "array"
+    },
+    "id": {
+      "pattern": "^\\$.+$",
```

Comment:
The engine does not require the id of an operator start with "$". Do we want to require it from the editors?

Reply:
Updated